lib/devise_security_extension/models/expirable.rb

```diff
@@ -20,15 +20,15 @@ module Expirable
 
       # Updates +last_activity_at+, called from a Warden::Manager.after_set_user hook.
       def update_last_activity!
-        self.update_column(:last_activity_at, Time.now.utc)
+        self.update_column(:last_activity_at, Time.zone.now)
```

Comment:
codeclimate, I forgot about the timezone myself and was just using `Time.now`. I went through the whole gem to replace them all. `Time.now.utc` is opinionated about how we should be setuping our servers. `Time.zone.now` should take the configured environment